go.mod

```diff
@@ -5,10 +5,10 @@ go 1.21
 toolchain go1.21.1
 
 require (
-	github.com/docker/buildx v0.12.2-0.20240220084849-89154c7d3303 // indirect
-	github.com/docker/cli v25.0.4-0.20240221083216-f67e569a8fb9+incompatible // indirect
+	github.com/docker/buildx v0.13.0 // indirect
+	github.com/docker/cli v26.0.0-rc1+incompatible // indirect
 	github.com/docker/compose/v2 v2.24.6 // indirect
 	github.com/docker/scout-cli v1.4.1 // indirect
-	github.com/moby/buildkit v0.13.0-beta3.0.20240201135300-d906167d0b34 // indirect
+	github.com/moby/buildkit v0.13.0 // indirect
 	github.com/moby/moby v25.0.3-0.20240203133757-341a7978a541+incompatible // indirect
```

Comment:
Why do we have a `github.com/moby/moby` vendored?

Reply:
it's mostly because the engine API docs are from there.